mods/cnc/languages/chrome/en.ftl

```diff
@@ -610,7 +613,7 @@ checkbox-perfgraph-container = Show Performance Graph
 checkbox-check-version-container = Check for Updates
 checkbox-perftext-container = Show Performance Text
 checkbox-sendsysinfo-container = Send System Information
-label-sendsysinfo-checkbox-container-desc = Your Operating System, OpenGL and .NET runtime versions, and language settings will be sent along with an anonymous ID to help prioritize future development.
+label-sendsysinfo-checkbox-container-desc = Your Operating System, OpenGL and .NET runtime versions, and language settings will be sent along with an anonymous ID to help prioritize future development
```

Comment:
full stop after sentence?

Reply:
Jovial will solve fullstops, out of the scope of this PR

Reply:
If it is out of scope, why are you touching them?

Reply:
Initially, I included full stops for all entries. However, abcdefg and Pun suggested removing them for single-line entries, which I did, except for the encyclopedia entries. Jovial wants to include them for missions and is willing to do so. Once he’s done, I’m happy to open this PR for full stop discussions. If you have a style guide for full stops, I can implement it.

Reply:
For clarity, it's not my intent to create a separate PR to restore mission/notification full stops. The idea seems odd.

Reply:
Originally I thought the full stop problem would be solved with your 100s of additions. A style guide for fullstops needs to be written on Discord that doesnt conflict with existing advice. I don’t want to make large changes for fullstops a third time.

Reply:
I suggest you use https://dictionary.cambridge.org/grammar/british-grammar/punctuation instead of a custom style guide defined by non-native speakers hidden in a proprietary chat service.